lighthouse-core/test/audits/seo/font-size-test.js

```diff
@@ -234,4 +237,54 @@ describe('SEO: Font size audit', () => {
     expect(auditResult.score).toBe(1);
     expect(auditResult.notApplicable).toBe(true);
   });
+
+  describe('attributes source location', () => {
+    async function runFontSizeAuditWithSingleFailingStyle(style, nodeProperties) {
+      const artifacts = {
+        URL: {finalUrl: 'http://www.example.com'},
```

Comment:
this is also not a valid `URL` artifact :)

Reply:
do we want to make artifacts in tests fully complete, or just the parts we need for the test?

Reply:
> do we want to make artifacts in tests fully complete, or just the parts we need for the test?

not necessarily, I just found it funny :)

(`URL` is declared at the top level, though, so could be reused like `validViewport` is)